tests/components/slide_local/test_config_flow.py

```diff
@@ -0,0 +1,346 @@
+        result["flow_id"],
+        {
+            CONF_HOST: HOST,
+            CONF_PASSWORD: "pwd",
+        },
+    )
+
+    assert result2["type"] is FlowResultType.CREATE_ENTRY
+    assert result2["title"] == HOST
+    assert result2["data"][CONF_HOST] == HOST
+    assert result2["data"][CONF_PASSWORD] == "pwd"
+    assert result2["data"][CONF_API_VERSION] == 1
+    assert not result2["options"][CONF_INVERT_POSITION]
+    assert len(mock_setup_entry.mock_calls) == 1
+
+
+async def test_user_api_error(
+    hass: HomeAssistant,
+    mock_slide_api: AsyncMock,
+    mock_setup_entry: AsyncMock,
+) -> None:
+    """Test we get the form."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] is FlowResultType.FORM
+    assert result["errors"] == {}
+
+    mock_slide_api.slide_info.return_value = None
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_HOST: HOST,
+            CONF_PASSWORD: "pwd",
+        },
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "user"
+    assert result["errors"]["base"] == "unknown"
+
+
+@pytest.mark.parametrize(
+    ("exception", "error"),
+    [
+        (ClientConnectionError, "cannot_connect"),
+        (ClientTimeoutError, "cannot_connect"),
+        (AuthenticationFailed, "invalid_auth"),
+        (DigestAuthCalcError, "invalid_auth"),
+        (Exception, "unknown"),
+    ],
+)
+async def test_api_1_exceptions(
+    hass: HomeAssistant,
+    exception: Exception,
+    error: str,
+    mock_slide_api: AsyncMock,
+) -> None:
+    """Test we can handle Form exceptions for api 1."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] is FlowResultType.FORM
+    assert result["errors"] == {}
+
+    mock_slide_api.slide_info.side_effect = [None, exception]
+
+    # tests with connection error
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_HOST: HOST,
+            CONF_PASSWORD: "pwd",
+        },
+    )
+
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "user"
+    assert result["errors"]["base"] == error
+
+    # tests with all provided
+    mock_slide_api.slide_info.side_effect = None
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {
+            CONF_HOST: HOST,
+            CONF_PASSWORD: "pwd",
+        },
+    )
+
+    assert result["type"] is FlowResultType.CREATE_ENTRY
+    assert result["title"] == HOST
+    assert result["data"][CONF_HOST] == HOST
+    assert result["data"][CONF_PASSWORD] == "pwd"
+    assert result["data"][CONF_API_VERSION] == 2
+    assert not result["options"][CONF_INVERT_POSITION]
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.CREATE_ENTRY
```
With the exception tests I think we don't have to add the full assertion as we already do that in other tests